Security/LoginManager.php

```diff
@@ -27,16 +28,19 @@
  */
 class LoginManager implements LoginManagerInterface
 {
-    private $securityContext;
+    /**
+     * @var SecurityContextInterface|TokenStorageInterface
+     */
+    private $tokenStorage;
     private $userChecker;
     private $sessionStrategy;
     private $container;
 
-    public function __construct(SecurityContextInterface $context, UserCheckerInterface $userChecker,
+    public function __construct($tokenStorage, UserCheckerInterface $userChecker,
                                 SessionAuthenticationStrategyInterface $sessionStrategy,
                                 ContainerInterface $container)
     {
```

Comment:
I don't find the place where we used it (components have bumped their interdependencies, and I cannot find the bundle where we used it). Here is what I'm asking for:

```php
if (!$tokenStorage instanceof TokenStorageInterface && !$tokenStorage instanceof SecurityContextInterface) {
    throw new \InvalidArgumentException('...');
}
```